Pages/3Design/3.1.0Architecture.md

```diff
@@ -37,43 +35,42 @@ The ViewModel mediates between the Model and the View. It exposes data and comma
 
 ## 3. Infrastructure
 
 ### 3.1 Operating Systems
 
-To run the software on the end-users system `Docker` using WSL2 as virtualization (assuming that the user uses Windows 10/11). No further software should be necessary to install. WSL2 already has the basic capability build in to run X11-aware applications.
+As outlined in [1. HMI](#1-hmi) the software was originally intended to run on every operating system without additional installation steps or dependencies but this did not work due to the GOCV library dependency not being able to link statically.
+
+To solve the compatibility issues and to make running the final product as easy as possible `Docker` was choosen as an alternative, the image providing the necessary dependencies and abstracting the users operating system.
+
+The most likely enduser operating system is windows as this is the standard in german university administration. Unfortunately additional steps are necessary. To run `Docker` on this platform with the GUI working  **WSL2** is also required. No further software should be necessary to install. WSL2 already has the basic capability build in to run X11-aware applications.
 
 ### 3.2 Programming Language
 
-After we tested different programming languages and their compatibility to different frontend tools in the exploration sprints, we decided to use the programming language Go and combine it with the toolkit Fyne. The reasons for that are described in the subdivision [1.6.0 Exploration Sprints](/3rdTry/1PlanningPhase/1.6.0ExplorationSprints.md).
+After we tested different programming languages and their compatibility to different frontend tools in the exploration sprints, we decided to use the programming language Go and combine it with the toolkit Fyne. The reasons for that are described in the subdivision [0.0.0 Research Phase](../0Research/0.0.0ResearchPhase.md).
 
 ### 3.3 Libraries
 
 ### Go Standard Library
 
-The Go Standard Library is structured in a modular fashion with each package addressing specific functionalities in a minimalist way. This enables the code's code organization, reusability and extensibility. As YAAC's programming language is Go, this library is used for many functions in the code.
+The full list of libraries and frameworks used in the project can be viewed in the [go_mod file](https://github.com/DHBW-SE-2023/YAAC/blob/master/go.mod). The most important libraries however are:
 
-### Tesseract
```

Comment:
I'd add the tesseract paragraph to the Wiki- it is a library that we use in our project

Reply:
Thesseract is described in 3.3 at gosseract including the description you provided yesterday

Reply:
found it- sorry. Looks good then!